tests/DCPS/Compiler/xcdr/xcdr.cpp

```diff
@@ -172,7 +252,7 @@ void baseline_checks(const Encoding& encoding, const DataView& expected_cdr)
   Type value;
   EXPECT_EQ(serialized_size(encoding, value), expected_cdr.size);
   // TODO(iguessthislldo): Does not work for XCDR1 and XCDR2 Mutable and XCDR2 Appendable
-  /* EXPECT_EQ(max_serialized_size(encoding, value), expected_cdr.size); */
+  // EXPECT_EQ(max_serialized_size(encoding, value), expected_cdr.size);
```

Comment:
I'm going to be fixing it in #1846 so please undo this to avoid a conflict.

Reply:
Yeah, this was done for easily commenting out other tests when implementing LC test. Forgot to put back when done.